default.json

```diff
@@ -336,28 +336,6 @@
       "matchManagers": [
         "github-actions"
       ],
-      "matchPackageNames": [
-        "actions/**",
-        "github/codeql-action/**"
-      ],
-      "matchUpdateTypes": [
-        "major",
-        "minor"
-      ],
-      "pinDigests": true
-    },
-    {
-      "matchManagers": [
-        "github-actions"
-      ],
-      "matchPackageNames": [
-        "!actions/**",
-        "!github/codeql-action/**"
-      ],
-      "matchUpdateTypes": [
-        "major",
-        "minor"
-      ],
```

Comment:
Not sure we want to remove the update types restriction. But given that we will be grouping all bumps into a single PR, that's less of an issue.

Reply:
This is a great point - @tomleb it sounds like without the major/minor part GHAs could be updated to non-stable tags. It seems like there could be some trade-offs because of that. I wonder if we can define it as 3 blocks and have it work the way we want?

1. All GHA updates grouped by manager (and pin digest?),
2. specific package names only to restrict to major/minor,
3. the exclude one to catch all (account for updates in approved workflows that don't tag releases ever)?

Reply:
+1 on breaking this down to accommodate all scenarios. We could merge as-is, give it a try and then amend on a follow-up PR. WDYT @tomleb @mallardduck?

Reply:
> The main concern here being proposing PRs to bumps that are not part of an actual release.

Hum.. It feels like renovate should do the right thing by default here where:

1. if there are tags, use the tags
2. if there are no tags, use the latest git commit

For example, say you have `docker/setup-qemu-action` at version `3.1.0` and there are new commits since that version, then renovate will not try to update it, but if there is a new version `3.2.0` tag or whatever, it will update.

No?

Reply:
(I haven't seen an update from renovate go from a tagged version to a non-tagged version)

Reply:
I'm not completely sure that the bumping operates on the exact same manner across different managers. But I agree that Renovate should do the right thing is most cases. We can definitely review any edge cases as and when.

Reply:
Alright then how about we give this a try as is as you suggest Paulo? :+1:

Reply:
Sorry been busy with meetings. 👍  from me on merging and trying it out.